### ISSUE TITLE
Mandatory 0x0F byte missing in Opcodes like RDTSC

### ISSUE THREAD

**Issue by ghost** [astocko/json-x86-64#1]
The RDSTC opcode(and a lot more opcodes) uses a 0x0F byte before the actual opcode, how can I read it?

**Comment by dlandahl** [comment]
Yep it looks like in the original source XML the escape bytes are there, but when they generated the JSON file they only used the final byte in the opcode sequence, because they're each just called "opcode". PLEASE FIX! It makes this otherwise very useful document useless.
